google/open-location-code
# Move "wiki" into repo

This PR moves the wiki into the repo. Wikis as implemented in GitHub do not allow PRs.

That means that after the change:

* Contributors can make pull requests against these pages, e.g. which is the recommended way to link to external implementations.

* We can make changes to the (currently) "wiki" file and the other files at the same time, which is needed to make a [complete specification](https://github.com/google/open-location-code/pull/463).

# How I made this PR

1. Clone open-location-code and open-location-code.wiki
2. Move the wiki inside docs/ in this repo
3. Fix all links

I did NOT make other changes. This is a pure-play pull request. Existing typos, errors are all kept.

# How to review and accept this PR

1. Review the files changed link above
2. Click some of the files, check some of the links
3. Merge PR
4. Disable the wiki feature for this repo in project setting (link at top of this page)

# Follow on work

1. After this PR is accepted, I will contribute additional PRs to text that was previously in the "wiki".
2. I will invite the PHP implementation to add to our list of external implementations https://github.com/google/open-location-code/issues/427
